### PR TITLE
Changelog updates for 48.21.0

### PR DESCRIPTION
**Pull request by sfsholden**
### What does this PR do?
Changelog for release 48.21.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+## Added
+
+#### docs
+
+- Japanese translations for documentation and messages. ([PR #2246](https://github.com/forcedotcom/salesforcedx-vscode/pull/2246)) -Contribution by [shunkosa](https://github.com/shunkosa)
+
+#### salesforcedx-vscode-core
+
+- changes for release ([PR #2270](https://github.com/forcedotcom/salesforcedx-vscode/pull/2270))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]: I wasn't sure what we wanted to include here, if anything, for the source-deploy-retrieve library.

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:11]: We do, but we need to word it as supporting Aura and LWC for the performance enhancement stuff cc @brpowell @amandastern99

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+# 48.21.0 - June 18, 2020
+
+## Added
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:3]: @sfsholden we can now mention the work on Mobile Preview for LWC components https://github.com/forcedotcom/salesforcedx-vscode/pull/2266

Comment (sfsholden) [reply packages/salesforcedx-vscode/CHANGELOG.md:3]: @lcampos Cool, I will add that entry back in.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -6,9 +6,11 @@
 
 - Japanese translations for documentation and messages. ([PR #2246](https://github.com/forcedotcom/salesforcedx-vscode/pull/2246)) -Contribution by [shunkosa](https://github.com/shunkosa)
 
+- Added Mobile Preview Selection for `SFDX: Preview Component Locally`. See [Local Development (Beta)](https://developer.salesforce.com/tools/vscode/en/lwc/local-dev) for more details. ([PR #2266](https://github.com/forcedotcom/salesforcedx-vscode/pull/2266)) -Contribution by [brandonpage](https://github.com/brandonpage)
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:9]:
```suggestion
- Added Mobile Preview Selection for `SFDX: Preview Component Locally`. See [Local Development (Beta)](https://developer.salesforce.com/tools/vscode/en/lwc/local-dev) for more details. ([PR #2266](https://github.com/forcedotcom/salesforcedx-vscode/pull/2266))
```
Brandon is part of salesforce so we don't need to add the contribution text

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
+
+#### docs
+
+- Japanese translations for documentation and messages. ([PR #2246](https://github.com/forcedotcom/salesforcedx-vscode/pull/2246)) -Contribution by [shunkosa](https://github.com/shunkosa)
+
+- Added Mobile Preview Selection for `SFDX: Preview Component Locally`. See [Local Development (Beta)](https://developer.salesforce.com/tools/vscode/en/lwc/local-dev) for more details. ([PR #2266](https://github.com/forcedotcom/salesforcedx-vscode/pull/2266)) -Contribution by [brandonpage](https://github.com/brandonpage)
+
+#### salesforcedx-vscode-core
+
+- Added support for Aura and LWC bundle types for beta deploys ([PR #2270](https://github.com/forcedotcom/salesforcedx-vscode/pull/2270))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]:
```suggestion
- Support for Aura and LWC deploys in [Performance Enhancements](https://developer.salesforce.com/tools/vscode/en/user-guide/perf-enhancements) ([PR #2270](https://github.com/forcedotcom/salesforcedx-vscode/pull/2270))
```

or something like that ? I also noticed we didn't update that doc file so I'll open a PR for it in a min

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,17 @@
+# 48.21.0 - June 18, 2020
+
+## Added
+
+#### docs
+
+- Japanese translations for documentation and messages. ([PR #2246](https://github.com/forcedotcom/salesforcedx-vscode/pull/2246)) -Contribution by [shunkosa](https://github.com/shunkosa)
+
+- Added Mobile Preview Selection for `SFDX: Preview Component Locally`. See [Local Development (Beta)](https://developer.salesforce.com/tools/vscode/en/lwc/local-dev) for more details. ([PR #2266](https://github.com/forcedotcom/salesforcedx-vscode/pull/2266))
```
Comment (jag-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:9]: I'm getting a 404 on the 'Local Development (Beta)' link. Can you please check.
Why is this in docs? Shouldn't it be under Added -> salesforcedx-vscode-lwc ?

**Pull request merged**